socorro/cron/jobs/bugzilla.py

```diff
@@ -56,17 +56,23 @@ class BugzillaCronApp(BaseCronApp):
         'days_into_past',
         default=0,
         doc='number of days to look into the past for bugs (0 - use last '
-            'run time)')
+            'run time, >0 ignore when it last ran successfully)')
 
     def run(self):
-        # record_associations
-        try:
-            # KeyError if it's never run successfully
-            # TypeError if self.job_information is None
-            last_run = self.job_information['last_success']
-        except (KeyError, TypeError):
-            last_run = (datetime.datetime.now(tz.gettz('UTC')) -
-                        datetime.timedelta(days=self.config.days_into_past))
+        # if this is non-zero, we use it.
+        if self.config.days_into_past:
+            last_run = (
+                datetime.datetime.now(tz.gettz('UTC')) -
```

Comment:
Used a lot in the backend code,  there exists this:
```python
from socorro.lib.datetime_util import utc_now
```

it makes a simpler looking line...